test/get.coffee

```diff
@@ -304,3 +304,28 @@ describe 'quest', ->
         quest options, (err, resp, body) ->
           assert.ifError err
           done()
+
+      it 'is thenable on success', (done) ->
+        @timeout 20000
+        options =
+          uri: "#{protocol}://httpbin.org/get"
+        quest options
+          .then (resp) ->
```

Comment:
How I'm reading this is that `then` is a function that takes in two arguments, both of which are functions as well. Is that correct?